docs/integrations/saas-cloud/druva.md

```diff
@@ -0,0 +1,117 @@
+1. Click **Next**.
+
+Once an app is installed, it will appear in your **Personal** folder, or other folder that you specified. You can share it with your organization.
+
+The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but with some patience, you will be able to view full graphs and maps.
+
+## Viewing Druva dashboards
+
+* All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
+
+ You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.
+
+* Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
+
+### Druva Overview dashboard
+
+**Druva - Overview**. The Overview dashboard in the Druva App provides a comprehensive view of the devices and cloud apps used by your organization. It includes widgets for unique user activities, device and app distribution, top users and administrators' details by device and event counts, and the trend of devices and apps over time. This dashboard is a powerful tool for monitoring activity, with the ability to detect and address potential security threats through the "failed events over time" widget. The "recent events details" widget offers a quick snapshot of the latest events. By using this dashboard, you can identify potential issues or anomalies, track changes in device and app usage, and gain a better understanding of events occurring in your organization's devices and cloud apps.<br/><img src={useBaseUrl('img/integrations/saas-cloud/druva-overview.png')} alt="druva-overview" width="750"/>
+
+### Druva Alerts Overview dashboard
+
+**Druva - Alerts Overview**. The Alerts dashboard in the Druva App provides comprehensive widgets to analyze security threats and take proactive measures. It includes widgets for tracking alerts by location, severity, and frequency over time, as well as detailed information on unusual activity events and the top users generating alerts. The recent alerts widget provides a quick snapshot of the latest security activity. By using this dashboard, organizations can identify and respond to potential threats efficiently, safeguarding their data effectively.<br/><img src={useBaseUrl('img/integrations/saas-cloud/druva-alerts-overview.png')} alt="druva-alerts-overview" width="750"/>
+
+### Druva Backup and Restore dashboard
```

Comment:
Druva - Backup and Restore Overview

Reply:
Please refer to the above comment for this as well.